firmware/robot2015/src-fpga/src/qencoder/README.md

```diff
@@ -0,0 +1,3 @@
+# MyHDL
+
```

Comment:
I was seeing how different it was writing HDL things in python using the MyHDL library. MyHDL is pretty cool and has pretty decent unit testing support, but I have already gotten use to Verilog. So I just left it in everything for reference.